.buckconfig

```diff
@@ -0,0 +1,6 @@
+
+[android]
```

Comment:
I guess this new file is coming from the react-native update. Don't know if we use this buck build system from facebook